### PR TITLE
MAINT / CI: add missing imports flagged by pylint; Fixes Issue #1904.

### LINKED ISSUE #1904: BLD / CI: pylint test failing
Looks like the pylint CI test is failing as of late -- this can be seen in the [CI of the recent compound PR](https://travis-ci.com/MDAnalysis/mdanalysis/jobs/124704700) and similarly in my appveyor PR.

This should be a straightforward fix -- I'll assign myself.

### PR DESCRIPTION
**Pull request by tylerjereddy**
Fixes #1904 

I tested locally with pylint & our pylintrc file & it was happy -- hopefully the genuine CI will also be happy.

### CODE REVIEW

**Comment by tylerjereddy** [comment]
It looks like pylint is reporting new issues after the old ones were patched, strange. Maybe the log gets truncated or something.

**Comment by orbeckst** [comment]
I had the same thing in PMDA cropping up suddenly MDAnalysis/pmda@80a0319640a721e04617810628057ed54324ad7d  (with the same annoying issue that not all problems were flagged together).

**Comment by orbeckst** [comment]
I'll review when Travis is happy, just ping me, please.

File: testsuite/MDAnalysisTests/analysis/test_wbridge.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ def test_count_by_type_multiple_frame(self):
         wb = WaterBridgeAnalysis(u, 'protein and (resid 1)', 'protein and (resid 4)')
         # Build an dummy WaterBridgeAnalysis object for testing
         wb._timeseries = True
-        wb.timesteps = range(6)
+        wb.timesteps = [0, 1, 2, 3, 4, 5]
```
Comment (kain88-de) [inline testsuite/MDAnalysisTests/analysis/test_wbridge.py]: do you still know the error that was thrown? I don't get any errors on test with the newest pylint and the develop branch.

The code itself is equivalent.

**Pull request merged**